book.rb

```diff
@@ -0,0 +1,15 @@
+# class book
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(rental)
+    @rental.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] Nice attempt trying to implement the many-to-many relationship between the `Book` class and the `Person` class through the `Rental` class. However, for this relationship to be properly implemented, the `Book` and `Person` classes need to have their `add_rental` method enable them to create a new `Rental` object instead of pushing the rental object to the `rentals` array. 👍 Kindly change what this method does. You can read [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) to have a better understanding. 👍